docs/references/files/fossa-yml.md

```diff
@@ -2,6 +2,8 @@
 
 The fossa configuration file, `.fossa.yml`, is an optional file located at the root of a project that can be used to configure project settings. 
 
+You can use [this configuration generator tool](https://wirechunk.com/topics/cf5f5c70-bfd3-48d9-9f3c-928dd42f50ef) to easily create a configuration file.
```

Comment:
Is this guaranteed to stay up to date with schema from repository?

Reply:
Soon enough there will be functionality to automatically maintain the generator (Wirechunk template) and the schema in this repo in sync. For now it'd require manual updates.